sdk/spring/azure-spring-cloud-context/src/main/java/com/azure/spring/cloud/autoconfigure/context/AzureContextAutoConfiguration.java

```diff
@@ -25,7 +26,9 @@
 @Configuration
 @EnableConfigurationProperties(AzureProperties.class)
 @ConditionalOnClass(AzureResourceManager.class)
+// TODO (yiliu6) Can the property be changed to auto-created?
```

Comment:
What does `auto-created` mean? Does it mean use `AzureProperties`?

Reply:
We will reserve resource-group prop here as the condition